### ISSUE TITLE
用文档类屏幕照片跑模型生成全黑的图

### ISSUE THREAD

**Issue by EYcab** [zhenngbolun/Learnbale_Bandpass_Filter#13]
我之前已经调通你这边的模型并用来跑生成过程，我发现如果用 AIM2019 的测试集跑，生成的图片看起来挺理想的，但是当我用业务数据（文档类屏幕照片）来跑的时候就会生成全黑的图片了，这是个什么现象呢？

是不是模型不够鲁棒？

**Comment by zhenngbolun** [comment]
Perhaps the model weights trained on AIM2019 dataset is not robust enough to solve general demoireing problems because the AIM2019 dataset is a generated dataset (not a real dataset). Basing on our experience, the model weights trained on TIP2018 dataset will be more robust for real image demoireing.

However, as you say that the model outputs a black image seems impossible. Maybe you should check your code first. Please notice that the output of the model is image with the range of [0, 1]. I guess you didn't convert the image to [0, 255] and directly saved the image of [0, 1] to the disk.

**Comment by EYcab** [comment]
Indeed, I ran your codes via main_multiscale.py where already have  255.0 multiplied. That is why I don't think that messes it up.

![main_multiscale_issues](https://user-images.githubusercontent.com/11568206/123759454-803b7e80-d8f2-11eb-9f22-4cc1555b5261.png)

**Comment by zhenngbolun** [comment]
We may upload the model weights trained on TIP2018 dataset later. You can try with this model weights.

**Comment by EYcab** [comment]
Thank you in advance,@zhenngbolun, but I have not seen your updated model weights on TIP2018, have you uploaded them yet?

**Comment by daeing** [comment]
> IM2019 dataset is not robust enough to solve general demoireing problems because the AIM2019 dataset is a generated dataset (not a real dataset). Basing on our experience, the model weights trained on TIP2018 dataset will be more robust for real image demoireing.

I notice you said AIM2019 dataset is a generate dataset. Do you know how to generate this dataset?

**Comment by fxcdl** [comment]
HI, I test the model on AIM2019 also generated black image, and I find the model output the negative matrix，so after *255, it s turned all back.I think it s abnormal.
![7](https://user-images.githubusercontent.com/50817018/180114055-ae440231-140a-454b-b425-9e0704757cea.png)

**Comment by deo651** [comment]
> 我之前已经调通你这边的模型并用来跑生成过程，我发现如果用 AIM2019 的测试集跑，生成的图片看起来挺理想的，但是当我用业务数据（文档类屏幕照片）来跑的时候就会生成全黑的图片了，这是个什么现象呢？
> 
> 是不是模型不够鲁棒？

同，您解决了吗
